packages/client/hmi-client/src/components/workflow/tera-workflow-node.vue

```diff
@@ -76,6 +89,21 @@ onMounted(() => {
 	workflowNode.value.addEventListener('mouseup', stopDrag);
 });
 
+function selectPort(port: WorkflowPort) {
```

Comment:
I can connect multiple outputs with the same input - perhaps you can check if the input is available?

Reply:
Going to leave it for v1, yeah.